build/build-sdk-images/tool/base/Dockerfile

```diff
@@ -30,12 +30,26 @@ RUN git clone -b $GRPC_RELEASE_TAG --depth=5 https://github.com/grpc/grpc /var/l
     cd /var/local/git/grpc && \
     git submodule update --init && \
     echo "--- installing protobuf ---" && \
-    cd third_party/protobuf && \
-    ./autogen.sh && ./configure --enable-shared && \
+    cd third_party/protobuf/cmake && \
+    mkdir -p build && cd build && \
+    cmake -Dprotobuf_BUILD_TESTS=OFF \
+          -DBUILD_SHARED_LIBS=ON \
+          -DCMAKE_BUILD_TYPE=Release \
+           .. && \
```

Comment:
Changed to use cmake instead of make because the latest protobuf doesn't suppor make installing.
```
#7 651.7
#7 651.7 Installing via 'make' is no longer supported. Use cmake or bazel instead.
#7 651.7
#7 651.7 Please consult BUILDING.md to get more information.
#7 651.7
#7 651.7 make: *** [Makefile:694: stop] Error 1
```